intro-to-R-tidyverse/03-intro_to_tidyverse-live.Rmd

```diff
@@ -321,25 +321,26 @@ Like if we want to see the genes with the highest average expression?
 We can use the same function, but instead use the `desc()` function and now we 
 are using `avg_expression` column. 
 
-```{r}
+```{r arrange-desc}
 # arrange descending by avg_expression
-
+stats_df %>%
```

Comment:
Keeping this on purpose?

Reply:
For now, yes. It is much easier to reconsider these decisions in the future though!